src/websockets/binary_websockets.es6

```diff
@@ -11,29 +11,43 @@ let socket = null;
 let is_website_up = false;
 let queued_requests = {};
 
-const get_app_id = () => {
-   const app_ids = JSON.parse(app_ids_json);
-   const config = local_storage.get('config');
-   let token = (config && config.app_id) || '';
-
-   if(!token) { /* find the appropriate token */
-      const href = window.location.href;
-      for(const web_address in app_ids) {
-         if(href.lastIndexOf(web_address, 0) == 0) {
-            token = app_ids[web_address];
-            break;
-         }
+
+const get_app_default_appid = () => {
+  const app_ids = JSON.parse(app_ids_json);
+  const config = local_storage.get('config');
+  let token = (config && config.app_id) || '';
+
+  if(!token) { /* find the appropriate token */
+    const href = window.location.href;
+    for(const web_address in app_ids) {
+      if(href.lastIndexOf(web_address, 0) == 0) {
+        token = app_ids[web_address];
+        break;
       }
+    }
+  }
+  localStorage.setItem('config.default_app_id', token);
```

Comment:
@nazaninreihani In code above you are using `local_storage` and then here you are using `localStorage`?

Reply:
extension needs default_app_id for reset button, 
I have to change the oauth link config too, so I'll fix the localStorage instead of local_storage in that PR

Reply:
@nazaninreihani Couple of points - 
When rest is clicked, just delete the locally stored values and the default hard coded values in the code will take over. I am still not sure why we have to store the default values in local storage.

Using localStorage will cross link the values across applications. Meaning, if you store with config.something using localStorage in webtrader, it might be accessible in other applications. I am not sure if this is desired behavior. We use local_storage in webtrader which ensures that all settings in webtrader/beta or webtrader-live or other applications do not cross link themeselves. You can open applications panel in developer mode and see for yourself.

Reply:
we have so many default_app_id for beta versions and gh-pages of apps that may change in codes by a developer, so It cannot be hardcoded in extension's core codes, and extension needs to access them in local storage to avoid reset to the global default app id. this makes more sense especially in gh-pages, if the default_app_id I've set as a developer in codes is not available to extension it will reset to '1' in binary static for example instead of default app id I've set and makes conflicts between the app and the extension.

Reply:
@nazaninreihani ok, I think before I explain the second paragraph, I probably need to understand the chrome extension throughly. Please see my comment in slack. Let me first get this chrome extension working in my local before going further.